src/pykoala/plotting/utils.py

```diff
@@ -151,12 +156,12 @@ def colour_map(fig, ax, cblabel, data,
 
     return im, cb
 
-def fibre_map(fig, ax, cblabel, data, rss=None, fib_ra=None, fib_dec=None,
-              s=100, cmap=default_cmap, norm=None, cbax=None,
-              norm_interval=AsymmetricPercentileInterval,
-              interval_args={"lower_percentile": 1.0,
-                              "upper_percentile": 99.0},
-              stretch=PowerStretch, stretch_args={"a": 0.7}):
+def plot_fibres(fig, ax, rss=None, x=None, y=None,
```

Comment:
Originally, in tutorial 1, `fibre_map` uses parameter `fib_ra`, which is not present in `plot_fibres`. I suggest whether adding `fib_ra=None` in `plot_fibres` or remove the parameter in the tutorial.